catalog/app/embed/Embed.js

```diff
@@ -136,36 +138,40 @@ function BucketLayout({ bucket, children }) {
 }
 
 function useInit() {
+  const messageParent = ipc.useMessageParent()
```

Comment:
`messageToParent` or `messageFromParent` could be more convinient

Reply:
"message" here is a verb, i.e. "to message one's parent". maybe it makes sense to use some other wording here, e.g. "notify" or smth, to distinguish sent and received messages ("command" vs "notification")